src/components/App/EpisodesList/Show/Details/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
Are those details for Show or for Episode? Are you sure about nesting folders here?